docs/docs/company/team.md

```diff
@@ -15,7 +15,7 @@ As the first engineer, he helped to build and grow the open source project from
 
 ### [Liren Tu](https://www.linkedin.com/in/tuliren/)
 
-Liren is a top-notch engineering manager, architect, and builder. 
+Liren is a software engineer.
```

Comment:
Selling yourself short Liren. I had ChatGPT jazz it up and got each of our intro lines. I really think you should keep it.

Reply:
Alright. I won't remove all of it. Some of the description actually was incorrect. For example, I did not reduce the time to create API integrations by hours. That was Sherif and Alex G, who authored the no / low-code CDK.

Reply:
Oh I was basing that on "Introduced code template for data connector creation. Shortened the time to write a new connector from hours to seconds." from LinkedIn.